projects/ngx-maplibre-gl/src/lib/control/geolocate-control.directive.ts

```diff
@@ -1,69 +1,70 @@
 import {
-  AfterContentInit,
   Directive,
-  EventEmitter,
-  Host,
-  Input,
-  Output,
+  afterNextRender,
+  inject,
+  input,
+  output,
 } from '@angular/core';
-import { FitBoundsOptions, GeolocateControl } from 'maplibre-gl';
+import {
+  type FitBoundsOptions,
+  GeolocateControl,
+  GeolocateControlOptions,
+} from 'maplibre-gl';
 import { MapService } from '../map/map.service';
 import { ControlComponent } from './control.component';
-import { Position } from '../map/map.types';
+import type { Position } from '../map/map.types';
+import { keepAvailableObjectValues } from '../shared/utils/functions/object.fn';
 
 /**
  * `mglGeolocate` - a geolocate control directive
- * 
+ *
  * @category Directives
- * 
+ *
  * @see [Locate user](https://maplibre.org/ngx-maplibre-gl/demo/locate-user)
  * @see [GeolocateControl](https://maplibre.org/maplibre-gl-js/docs/API/classes/GeolocateControl)
  */
 @Directive({
   selector: '[mglGeolocate]',
   standalone: true,
 })
-export class GeolocateControlDirective implements AfterContentInit {
+export class GeolocateControlDirective {
+  /* Init injection */
+  private readonly mapService = inject(MapService);
+  private readonly controlComponent = inject<
+    ControlComponent<GeolocateControl>
+  >(ControlComponent, { host: true });
+
   /* Init inputs */
-  @Input() positionOptions?: PositionOptions;
-  @Input() fitBoundsOptions?: FitBoundsOptions;
-  @Input() trackUserLocation?: boolean;
-  @Input() showUserLocation?: boolean;
+  readonly positionOptions = input<PositionOptions>();
+  readonly fitBoundsOptions = input<FitBoundsOptions>();
```

Comment:
Missing `/** Init input */` here too, although it was missing before this change as well...